olive/platform_sdk/qualcomm/utils/data_loader.py

```diff
@@ -128,38 +120,54 @@ def get_input_list(self):
 
 
 class FileListProcessedDataLoader(FileListDataLoader):
+    """FileList dataloader created from directory with data processed as expected by the Qualcomm SDKs."""
+
     def __init__(
         self,
         data_dir: str,
         input_list_file: str = "input_list.txt",
         annotations_file: str = None,
         batch_size: int = None,
     ):
-        config = {"data_dir": data_dir, "input_list_file": input_list_file, "annotations_file": annotations_file}
-        super().__init__(config, batch_size)
+        """Initialize FileListProcessedDataLoader.
 
-    def load_data(self) -> Tuple[str, str, np.ndarray]:
-        self.tmp_dir = tempfile.TemporaryDirectory(prefix="olive_tmp_")  # pylint: disable=consider-using-with
-        input_list = input_list_utils.get_input_list(
-            self.config["data_dir"], self.config["input_list_file"], self.tmp_dir.name
+        :param data_dir: directory containing processed data.
+        :param input_list_file: file containing input list. The paths to inputs are expected to be relative to data_dir.
+        :param annotations_file: npy file containing annotations. If None, no annotations are loaded.
+        :param batch_size: number of inputs per chunked input list file for batch processing. If None, all inputs are in
+            a single input list file.
+        """
+        super().__init__(
+            self.load_data,
```

Comment:
`FileListProcessedDataLoader.load_data` for static function calling?

Reply:
keep as is for now since `self.load_data` here also refers to the same function.